### ISSUE TITLE
Generator runs only once per iteration

### ISSUE THREAD

**Issue by VigneshSrinivasan10** [ikostrikov/TensorFlow-VAE-GAN-DRAW#3]
Hi,

Great work!

Usually in GAN, the generator is run twice to make sure that discriminator loss does not go to zero. And I noticed that you are not doing this in your implementation!

Thanks!

**Comment by ikostrikov** [comment]
Hi!
Thanks! I decided to control this with different learning rates instead as in:
http://arxiv.org/pdf/1511.06440.pdf

I found it slightly more stable.

**Comment by igul222** [comment]
@vignesh10 Could you provide a citation for this? So far I've (somewhat counterintuitively) had better results training the discriminator more frequently than the generator.

**Comment by VigneshSrinivasan10** [comment]
@igul222 , It is in general a rule of thumb. And as mentioned by @ikostrikov , it can also be controlled using different learning rates,
1. Section 6 in https://papers.nips.cc/paper/5423-generative-adversarial-nets.pdf discusses it. 
2. https://recordnotfound.com/DCGAN-tensorflow-carpedm20-13712 (not a citation!)
3. https://www.reddit.com/r/MachineLearning/comments/3vktp4/seq2seq_generative_adversarial_net/? (discussion)
